### PR TITLE
Refactor/#1433 Rename the classes and fileds

### PR DESCRIPTION
**Pull request by Taaaaaserface**
In this PR, I refactor the variable names with leading underscore by removing the underscore.

### CODE REVIEW

File: inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/trie/Trie.java [hunk 1]
```diff
@@ -429,34 +429,34 @@ private void keys(final Character c, final Node n, final StringBuilder b,
         {
             private final Character _c;
```
Comment (munterkalmsteiner) [inline inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/trie/Trie.java]: If you keep the underscore here, you make the refactoring inconsistent. You should get rid of all leading underscores.

File: inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/trie/Trie.java [hunk 2]
```diff
@@ -429,34 +429,34 @@ private void keys(final Character c, final Node n, final StringBuilder b,
...
             private final Node _n;
-            private final Iterator<Character> _i;
-            private boolean _nodeDone;
+            private final Iterator<Character> i;
+            private boolean nodeDone;
 
             public Frame(final Character c, final Node n)
```
Comment (munterkalmsteiner) [inline inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/trie/Trie.java]: You could rename `c` into `aChar`. Similar with `n`.

File: inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/util/OverlapIterator.java [hunk 3]
```diff
@@ -30,216 +30,216 @@
 
 public class OverlapIterator
 {
-    private final Logger _log = LoggerFactory.getLogger(getClass());
+    private final Logger log = LoggerFactory.getLogger(getClass());
 
     /** The A/B lists */
     private final List<Offset> _la;
```
Comment (munterkalmsteiner) [inline inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/util/OverlapIterator.java]: Same here. Be consistent.

File: inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/util/OverlapIterator.java [hunk 4]
```diff
@@ -30,216 +30,216 @@
...
 
-    private int _last_b_step_na;
-    private boolean _done;
+    private int last_b_step_na;
+    private boolean done;
 
-    private int _stepCount;
+    private int stepCount;
 
     public OverlapIterator(final List<Offset> la, final List<Offset> lb)
```
Comment (munterkalmsteiner) [inline inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/util/OverlapIterator.java]: Maybe call them `aList` and `bList`

File: inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/trie/Trie.java [hunk 5]
```diff
@@ -36,7 +36,7 @@
 public class Trie<V>
```
Comment (munterkalmsteiner) [inline inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/trie/Trie.java]: This class has still a method that starts with an underscore (`_getNode()`). I'd suggest to change that too.

File: inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/util/OverlapIterator.java [hunk 6]
```diff
@@ -30,216 +30,216 @@
...
             steppedBack = true;
         }
 
         // Correct pointer
         if (steppedBack) {
             // Make sure the next peek really peeks ahead.
-            _na = _ia.nextIndex();
-            _cura = _ia.next();
+            na = ia.nextIndex();
+            cura =ia.next();
```
Comment (munterkalmsteiner) [inline inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/util/OverlapIterator.java]: Space missing after `=`

**Comment by reckart** [comment]
Jenkins, can you test this please?

**Comment by reckart** [comment]
Looks good :) The commit comments to not conform to our conventions, so I'll perform a squash merge so I can easily edit them.

**Comment by reckart** [comment]
Thanks :)

**Comment by Taaaaaserface** [comment]
> Thanks :)

You are welcome <3

**Pull request merged**